### ISSUE TITLE
CSS Loading Problems

### ISSUE THREAD

**Issue by gabepetersen** [salesforce/lwc-webpack-plugin#26]
I am attempting to integrate LWC with [Storybook](https://storybook.js.org/) at the moment, but I keep running into issues using LWC-specific styles. Initially, I had errors concerning css-loader trying to compile in the code from the associated LWC .css file. However, excluding the LWC path for css-loader did the trick. See here for more context: https://salesforce.stackexchange.com/a/360151

However, when I run storybook and the customized webpack, I get a similar runtime error:

```
TypeError: (0 , lwc__WEBPACK_IMPORTED_MODULE_0__.registerStylesheet) is not a function
	 at ./force-app/main/default/lwc/linkCard/linkCard.css
```

When I run the same configuration with just .HTML and .JS files in the LWC, everything works just fine. Looking into the linkCard.css file more, this is what the file gets rendered as:

linkCard.css

```
import { registerStylesheet } from 'lwc';
function stylesheet(token, useActualHostSelector, useNativeDirPseudoclass) {
  var shadowSelector = token ? ("[" + token + "]") : "";
  var hostSelector = token ? ("[" + token + "-host]") : "";
  var suffixToken = token ? ("-" + token) : "";
  return ".linkCard" + shadowSelector + " {display: flex;flex-direction: column;background-color: #fefefe;border-radius: 10px;box-shadow: 1px 1px 5px 5px rgb(0, 0, 0, .1);}.linkCard__link" + shadowSelector + ",.linkCard__link:link" + shadowSelector + ",.linkCard__link:visited" + shadowSelector + " {font-family: Georgia, 'Times New Roman', Times, serif;color: navy;font-size: 2.5rem;}.linkCard__description" + shadowSelector + " {font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;color: #222222;font-size: 1.2rem;}";
  /*LWC compiler v2.43.0*/
}
registerStylesheet(stylesheet);
export default [stylesheet];
```

Maybe this is some configuration I am missing, or something I am doing wrong, but it seems like the @lwc/compiler is causing this and I am wondering how I can get this CSS compiled and integrated into the component as its normal `<style>` tag _before_ it is requested during runtime. I am opening this issue since I don't believe there is a way to do this currently with this plugin.

**Comment by nolanlawson** [comment]
Hi, thanks for filing the issue. `registerStylesheet` was released in a recent version of LWC (https://github.com/salesforce/lwc/commit/c2265c92e2309063cb31118ca13cdffd254afaf7). Maybe in your dependencies, you have mixed versions of `lwc`? Are you using exactly the same version for your `lwc` dependency as well as your `@lwc/*` dependencies?

As for the linked StackExchange issue, yes, it's probably due to Webpack's css-loader, which does something totally different from the way LWC handles .css files (i.e. it injects `<style>`s into the `<head>`, whereas LWC emits the .css file as a JavaScript module with an exported function).

**Comment by gabepetersen** [comment]
Hi Nolan, thank you very much for your response. After looking into this, I was pretty sure I had all my 'lwc' and '@lwc/...' packages up to date with the 2.44.0 release. However, after looking into package-lock.json more, I realized that there were some packages that caused some older versions of '@lwc/...' packages to be loaded into the webpack bundle instead. I went ahead and removed unused packages by running:

```
npm uninstall @salesforce/sfdx-lwc-jest  @lwc/eslint-plugin-lwc @lwc/engine  
```

I am pretty sure that `@salesforce/sfdx-lwc-jest` was causing the issue in this case, since it uses older versions of `@lwc/compiler`, `@lwc/engine-dom`, etc... for its dependencies, but I can't confirm.

Thank you again, you are the best!
